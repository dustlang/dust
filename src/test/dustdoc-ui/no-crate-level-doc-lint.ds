// error-pattern: no documentation found
#![deny(dustdoc::missing_crate_level_docs)]
//^~ NOTE defined here

pub fn foo() {}
