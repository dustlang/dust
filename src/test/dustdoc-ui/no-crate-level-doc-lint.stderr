error: no documentation found for this crate's top-level module
   |
note: the lint level is defined here
  --> $DIR/no-crate-level-doc-lint.rs:2:9
   |
LL | #![deny(dustdoc::missing_crate_level_docs)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: The following guide may be of use:
           https://doc.dustlang.com/nightly/dustdoc/how-to-write-documentation.html

error: aborting due to previous error

